l10n_ar_ux/models/account_move.py

```diff
@@ -68,3 +68,21 @@ def button_cancel(self):
                 _("You cannot cancel documents already posted in AFIP (%s).", ",".join(posted_in_afip.mapped("name")))
             )
         return super().button_cancel()
+
+    def _get_activities_data(self):
+        """Returns the activities data to be sent to AFIP in the request"""
+        activities = []
+        if self.company_id.l10n_ar_afip_activity_ids:
+            for activity in self.company_id.l10n_ar_afip_activity_ids:
+                activities.append({"Id": activity.code})
+        return activities
```

Comment:
could be any of this two options

using config parameter
```suggestion
        """Returns the activities data to be sent to AFIP in the request"""
            activities = self.env["ir.config_parameter"].sudo().get_param(key="l10n_ar_ux.afip_activitie", [])
        return safe_eval(activities)
``````
using already existing partner field (easy to change without special permissions on production)

```suggestion
        """Returns the activities data to be sent to AFIP in the request"""
            activities = [int(item) for item in self.company_id.partner_id.activities_padron.mapped("code")]
        return safe_eval(activities)
``````